src/schema/city/constants.ts

```diff
@@ -1 +1,2 @@
 export const LOCAL_DISCOVERY_RADIUS_KM = 75
+export const NEAREST_CITY_THRESHOLD_KM = 150
```

Comment:
I picked an arbitrary value here of 2x the city radius we had already defined. Both of these are tunable, of course